### PR TITLE
Apr 2226 create application withdrawn page seen by applicant

### PR DESCRIPTION
**Pull request by ConorTill**

### CODE REVIEW

**Review by MarkFCain — APPROVED** [review]

File: src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationWithdrawn.cshtml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    Layout = "_Layout";
+}
+
+<nav>
+    <ul id="navigation" class="roatp__navigation " aria-label="Top Level Navigation">
+        <li class="roatp__navigation-item">
+            <a class="govuk-link govuk-link--no-visited-state roatp__navigation-link--current">Application overview</a>
+        </li>
+        <li class="roatp__navigation-item roatp__navigation-item--signout">
+            <a class="govuk-link govuk-link--no-visited-state" asp-action="SignOut" asp-controller="Users">Save and sign out</a>
```
Comment (martin-lush) [inline src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationWithdrawn.cshtml:15]: It's part of the ticket but you can't Save anything on a Withdrawn application. Thus the content is incorrect for this application state. Can you query this with Design Team / Content.

File: src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationRejected.cshtml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    Layout = "_Layout";
+}
+
+<nav>
+    <ul id="navigation" class="roatp__navigation " aria-label="Top Level Navigation">
+        <li class="roatp__navigation-item">
+            <a class="govuk-link govuk-link--no-visited-state roatp__navigation-link--current">Application overview</a>
+        </li>
+        <li class="roatp__navigation-item roatp__navigation-item--signout">
+            <a class="govuk-link govuk-link--no-visited-state" asp-action="SignOut" asp-controller="Users">Save and sign out</a>
```
Comment (martin-lush) [inline src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationRejected.cshtml:15]: It's part of the ticket but you can't Save anything on a Removed application. Thus the content is incorrect for this application state. Can you query this with Design Team / Content.

File: src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationRejected.cshtml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        </li>
+    </ul>
+</nav>
+
+<main class="govuk-main-wrapper " id="main-content" role="main">
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <div class="govuk-panel app-panel--warning">
```
Comment (martin-lush) [inline src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationRejected.cshtml:25]: It's probably a no but... wouldn't it been better to have this as _govuk-panel--warning_ or make the ones that use _govuk-panel--confirmation_ have a named one like _app-panel--confirmation_ so at least all styles are consistently named for the panels

File: src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationWithdrawn.cshtml (hunk 1 above)
Comment (martin-lush) [reply src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationWithdrawn.cshtml]: spoke to Dale. Have agreed the updated content and put it into a partial view

File: src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationRejected.cshtml (hunk 2 above)
Comment (martin-lush) [reply src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationRejected.cshtml]: spoke to Dale. Have agreed the updated content and put it into a partial view

**Review by martin-lush — APPROVED** [review]

**Pull request merged**